File: particle-wallet/android/src/main/java/com/particlewallet/model/ChainData.java
package com.particlewallet.model;

import com.google.gson.annotations.SerializedName;

public class ChainData {

    @SerializedName("chain_name")
    public String chainName;

    @SerializedName("chain_id")
    public long chainId;

    @SerializedName("chain_id_name")
    public String chainIdName;

}

File: particle-biconomy/android/src/main/java/com/particlebiconomy/model/ChainData.kt
package com.particlebiconomy.model

import com.google.gson.annotations.SerializedName

class ChainData {
    @SerializedName("chain_name")
    var chainName: String? = null

    @SerializedName("chain_id")
    var chainId:Long = 0

    @SerializedName("chain_id_name")
    var chainIdName: String? = null

    @SerializedName("env")
    var env: String? = null
}
